### PR TITLE
fix(mae-consumer-docker): Fix condition for skipping elasticsearch check

### PR DESCRIPTION
**Pull request by dexter-mh-lee**
Mistakenly set it in the opposite way. check for elasticsearch if SKIP_ELASTICSEARCH_CHECK is not set or is false. 

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: ingestion-scheduler/src/main/java/com/datahub/metadata/ingestion/IngestionScheduler.java [hunk 1]
```diff
@@ -80,23 +82,18 @@
...
 
-  // Visible for testing.
-  public IngestionScheduler(@Nonnull final Authentication systemAuthentication,
-      @Nonnull final JavaEntityClient entityClient, @Nonnull final IngestionConfiguration configuration,
-      final int batchGetDelayIntervalSeconds, final int batchGetRefreshIntervalSeconds) {
-    _systemAuthentication = Objects.requireNonNull(systemAuthentication);
-    _entityClient = Objects.requireNonNull(entityClient);
-    _ingestionConfiguration = configuration;
-
+  public void init() {
```
Comment (jjoyce0510) [inline ingestion-scheduler/src/main/java/com/datahub/metadata/ingestion/IngestionScheduler.java:88]: nice!

File: metadata-jobs/mae-consumer/src/main/java/com/linkedin/metadata/kafka/MetadataChangeLogProcessor.java [hunk 2]
```diff
@@ -27,7 +28,7 @@
 @Slf4j
 @Component
 @Conditional(MetadataChangeLogProcessorCondition.class)
-@Import({ UpdateIndicesHook.class, IngestionSchedulerHook.class })
+@Import({ UpdateIndicesHook.class, IngestionSchedulerHook.class, KafkaEventConsumerFactory.class })
```
Comment (jjoyce0510) [inline metadata-jobs/mae-consumer/src/main/java/com/linkedin/metadata/kafka/MetadataChangeLogProcessor.java:31]: Funny that we had to add this...

**Review by jjoyce0510 — APPROVED** [review]

File: metadata-jobs/mae-consumer/src/main/java/com/linkedin/metadata/kafka/MetadataChangeLogProcessor.java (hunk 2 above)
Comment (dexter-mh-lee) [reply metadata-jobs/mae-consumer/src/main/java/com/linkedin/metadata/kafka/MetadataChangeLogProcessor.java:31]: It was there before, but was removed in the managed ingestion pr.

**Review by shirshanka — APPROVED** [review]
LGTM

**Pull request merged**